README.md

```diff
@@ -1,46 +1,90 @@
+<div align="center">
+
+![Top banner](/docs/images/banner.jpg)
+
+AutoKitteh is a **developer** platform for workflow automation and
+orchestration. It is an easy-to-use, code-based alternative to no/low-code
+platforms (such as Zapier, Workato, Make.com, n8n) with unlimited flexibility.
+
+In addition, it is a **durable execution** platform for long-running and
+reliable workflows. It is based on [Temporal](https://temporal.io/), hiding
+many of its infrastructure and coding complexities.
+
+AutoKitteh can be self-hosted, and has a cloud offering as well.
```

Comment:
We don't have cloud offering yet

Reply:
Checked with Haim. Given the location of this sentence (the first impression of the reader at the very top of the page), he prefers not to complicate this basic statement there. But we will add details/links somewhere, similar to how we're discussing the SaaS in the AK website, I guess.